binding/jms/src/test/java/io/tracee/binding/jms/TraceeMessageListenerTest.java

```diff
@@ -29,7 +28,8 @@
 	private final TraceeMessageListener unit = new TraceeMessageListener(backend);
 	private final InvocationContext invocationContext = mock(InvocationContext.class);
 	private final Message message = mock(Message.class);
-	private final Map<String,String> encodedContext = new HashMap<String, String>();
+	private final Map<String, String> encodedContext = new HashMap<String, String>();
+	private MdbLike mdbLike = spy(new MdbLike());
```

Comment:
Hmm, I think this isn't possible because the `backend` variable isn't accessable within a static class but maybe you know a good way. But I'm fine with `spy(new MdbLike())` as well. :-D